pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -794,6 +794,22 @@ final class Event {
           },
         );
 
+  /// Event that is sent from an IDE plugin.
+  ///
+  /// [name] - the name of the event.
+  ///
+  /// [ide] - the reporting [IDE].
```

Comment:
Technically, this isn't being reported by an IDE; it's being reported by an IDE plugin. (Or plugin pair; I don't think we want to differentiate between the Dart and Flutter plugins so that we can easily move functionality between them without complicating analytics processing.)

Also, when would this be different than the `DashTool` value (which is automatically reported by this package)? (I'm ignoring the fact that you're not differentiating between IntelliJ and Android Studio, because I think we need to do that, so that difference won't exist long term.)